Doc/source/install_linux.rst

```diff
@@ -64,8 +64,9 @@ To get the dependencies for building documentation::
 
 CDF
 ===
-If building SpacePy from source, you will need to install the NASA CDF
-library. This is not required if installing from a binary wheel.
+The NASA CDF library is included if installing from a binary wheel. If
+building SpacePy from source or wish to use a different version of the
```

Comment:
It looks like you either thought words and didn't type them, or rewrote something and lost words and a comma. Can we change to: "if building SpacePy from source, or if you wish to use a different version of the CDF library," (see also the equivalent spot in `install_mac.rst`

Reply:
Thanks, I just added a subject and pushed.